ccmlib/scylla_docker_cluster.py

```diff
@@ -39,6 +41,8 @@ def __init__(self, *args, **kwargs):
         self.docker_name = f'{self.cluster.get_path().split("/")[-2]}-{self.cluster.name}-{self.name}'
         self.jmx_port = "7199"  # The old CCM code expected to get a string and not int
         self.log_thread = None
+        self.share_directory = os.path.join(tempfile.mkdtemp(), os.path.split(self.get_path())[1])
```

Comment:
shouldn't it be inside the cluster or node configuration tree (under .ccm)

Reply:
fixed